pkg/discovery/worker/compliance/affinity_commodity_manager.go

```diff
@@ -0,0 +1,124 @@
+package compliance
+
+import (
+	"fmt"
+	"strconv"
+
+	api "k8s.io/client-go/pkg/api/v1"
+
+	sdkbuilder "github.com/turbonomic/turbo-go-sdk/pkg/builder"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+
+	"github.com/mitchellh/hashstructure"
+)
+
+const (
+	accessCommodityDefaultCapacity = 1E10
```

Comment:
As long as it is a large number, it is fine. AccessCommodity only acts as constraints and should not have an impact on the total money spent in market.